borg/archiver.py

```diff
@@ -537,7 +551,7 @@ def do_delete(self, args, repository):
             if not args.cache_only:
                 msg = []
                 try:
-                    manifest, key = Manifest.load(repository)
+                    manifest, key = Manifest.load(repository, Manifest.NO_OPERATION_CHECK)
```

Comment:
I meant the default on `Manifest.load` itself, though. Fair enough either way, you make a solid point.